framework/auth/core.py

```diff
@@ -977,6 +977,22 @@ def get_activity_points(self, db=None):
         db = db or framework.mongo.database
         return analytics.get_total_activity_count(self._primary_key, db=db)
 
+    def disable_user(self):
+        import mailchimp
```

Comment:
You should be able to import this at the top of the file. Follow the COSDev guidelines for import order.